packages/core/upload/admin/src/pages/App/components/Search.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
```

Comment:
You can use the Search component from the helper plugin instead see the comment below.